# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/webkitgtk/x32.diff
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- webkitgtk-2.36.0/Source/JavaScriptCore/CMakeLists.txt.vanilla	2022-04-24 00:18:45.393958595 +0200
+++ webkitgtk-2.36.0/Source/JavaScriptCore/CMakeLists.txt	2022-04-24 00:19:42.010956338 +0200
@@ -261,9 +261,9 @@
 if (WIN32)
     set(OFFLINE_ASM_BACKEND "X86_WIN, X86_64_WIN, C_LOOP_WIN")
 else ()
-    if (WTF_CPU_X86)
+    if (WTF_CPU_X86X)
         set(OFFLINE_ASM_BACKEND "X86")
-    elseif (WTF_CPU_X86_64)
+    elseif (WTF_CPU_X86_64X)
         set(OFFLINE_ASM_BACKEND "X86_64")
     elseif (WTF_CPU_ARM64)
         set(OFFLINE_ASM_BACKEND "ARM64")
@@ -1412,7 +1412,7 @@
 
 list(APPEND JavaScriptCore_HEADERS ${JavaScriptCore_DERIVED_SOURCES_DIR}/InjectedScriptSource.h)
 
-if (WTF_CPU_X86_64)
+if (WTF_CPU_X86_64X)
     if (MSVC AND NOT ENABLE_C_LOOP)
         add_custom_command(
             OUTPUT ${JavaScriptCore_DERIVED_SOURCES_DIR}/JITStubsMSVC64.obj
--- webkitgtk-2.36.0/Source/cmake/WebKitFeatures.cmake.vanilla	2022-04-24 17:59:26.882421026 +0200
+++ webkitgtk-2.36.0/Source/cmake/WebKitFeatures.cmake	2022-04-24 18:00:53.320417579 +0200
@@ -62,7 +62,7 @@
 macro(WEBKIT_OPTION_BEGIN)
     set(_SETTING_WEBKIT_OPTIONS TRUE)
 
-    if (WTF_CPU_ARM64 OR WTF_CPU_X86_64)
+    if (WTF_CPU_ARM64 OR WTF_CPU_X86_64X)
         set(ENABLE_JIT_DEFAULT ON)
         set(ENABLE_FTL_DEFAULT ON)
         set(USE_SYSTEM_MALLOC_DEFAULT OFF)
